Code/GraphMol/Bond.h

```diff
@@ -98,8 +98,10 @@ class Bond : public RDProps {
     STEREOANY,       // intentionally unspecified
```

Comment:
but aren't we guaranteed that the values in an enum will be assigned increasing values?